docs/builder.rst

```diff
@@ -46,13 +55,26 @@ The builder (i.e. ``build()``) expects a dict as the first argument. This dict c
    >>> assert built_manifest == expected_manifest
    >>> assert extended_manifest == expected_manifest
 
+With ``init_manifest()``, which populates "version" with "2" (the only supported EthPM specification version), unless provided with an alternative "version".
+
+.. doctest::
+
+   >>> initialized_manifest = build(
+   ...     init_manifest("owned", "1.0.0"),
+   ... )
+   >>> assert initialized_manifest == expected_manifest
+
+
 
 To return a ``Package``
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code:: python
 
-   return_package(w3: Web3)
+   build(
+       ...,
+       return_package(w3: Web3),
```

Comment:
maybe rename to `as_package`?  shorter and doesn't have the `return` keyword in it which can be visually confusing.